client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,99 @@
+import { Grid, Box, Typography, Rating, Divider } from '@mui/material';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import AvatarDisplay from '../../components/AvatarDisplay/AvatarDisplay';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import RequestForm from './RequestFormSubmit/RequestFormSubmit';
+import useStyles from './useStyles';
+import { useEffect, useState } from 'react';
+
+const testProfileDetail = {
+  name: 'grey naki',
+  description: 'pet lover',
+  location: 'Vancouver, Canada',
+  introduction:
+    'I grew up with animals my whole life. I have a passion for pets of all sizes and species. I am excited to be able to help you by caring for your pets while you are away. Your babies will be cared for and loved just as if they were my own.',
+  hourlyRate: 16,
+  photoUrl: 'https://images-na.ssl-images-amazon.com/images/I/61HtctiMZqL.jpg',
+  rating: 3.7,
+};
+
+export default function ProfileDetails(): JSX.Element {
+  const classes = useStyles();
+
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+
+  const updateWindowWidth = () => {
+    setWindowWidth(window.innerWidth);
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', updateWindowWidth);
+    return () => window.removeEventListener('resize', updateWindowWidth);
+  });
```

Comment:
It looks like this code is used for resizing page components based on hardcoded breakpoints. Consider using standardized breakpoints provided with mui.